packages/e2e/tests/proofGeneration.test.ts

```diff
@@ -1,3 +1,5 @@
+import { SemaphoreFullProof, RLNSNARKProof } from "@cryptkeeperzk/types";
```

Comment:
Why do you need types for tests?

Reply:
You can cast those objects to `Record<string, unknown>` to get rid of these errors.